draft for approval (#53)
PQCA/TAC: meeting-minutes/2024/2024-09-25-TAC.md

## Voting Representative Attendance (_Alphabetical by 1st name_)
### Premier Member Representatives
* [x] Brian Jarvis, Amazon Web Services Inc. [TAC Vice Chair]
* [x] Michael (Max)imilien, IBM [TAC Chair]
* [x] Norman Ashley, Cisco
* [ ] Sophie Schmieg, Google
* [x] Yarkin Doroz, NVIDIA
 ### Project Representatives
* [x] Nigel Jones, IBM [PQCP]
* [ ] TBD [OQS]
## Non-Voting Representative Attendance
### LF Staff
* [x] Hart Montgomery
* [ ] Ry Jones
* [x] Kenny Paul
### Other Attendees
* Alex Bozarth
* Bryan Uhri | Keyfactor
* Jason Lee - GSA
* JP Lomas (Die QRL Stiftung)
* Nicklas Körtge
* Basil Hess

## Agenda
Added project updates
* Introduction (as needed) (all)
* OQS

> * PQPC

Reviewer: PQPC --> PQCP

> * PQPC

Reviewer 2: ```suggestion
* PQCP
```

* ICMC PQ Day and Happy Hour recap (all)
* Tooling WG vote results and next steps (all, Nicklas)
* AWS QKD (Brian)
* Lifecycle doc pending issues (all)
* Any New business
* Action item review
---
### OQS update (Basil)
* liboqs release

> * Implementation of kyber (sp) & mlkem(sp)

Reviewer: Kyber, ML-KEM

> * Implementation of kyber (sp) & mlkem(sp)

Reviewer 2: ```suggestion
* Implementation of Kyber & ML-KEM
```

* Signature keys - disabled by default

> * Many non functional improvemens like moving things to GHI

Reviewer: improvements

> * Many non functional improvemens like moving things to GHI

Reviewer 2: ```suggestion
* Many non functional improvements like moving things to GHI
```

* OpenSSL provider being updated
--
* other project work in progress
* Waiting for liboqs update for interoperation 
* Discussion now turning to a V1 release.
___
### PQCP update (Nigel)
* TSC meeting tomorrow

> * libjade(sp) looking to implementation for alignment

Reviewer: Spelling is correct

> * libjade(sp) looking to implementation for alignment

Reviewer 2: ```suggestion
* libjade looking to implementation for alignment
```

> * PR open for krystals(sp) please look at the pr

Reviewer: Assuming this is referring to CRYSTALS. ([ref](https://pq-crystals.org))

> * PR open for krystals(sp) please look at the pr

Reviewer 2: ```suggestion
* PR open for merging PQCrystals code : please look at the pr in mlkem-c-generic
```

* embedded teams have made a lot of progress
___
### ICMP Happy Hour
* The PQCA happy hour was well attended with good interaction -  was a nice end to PQ day
* Creating an awareness of PQCA was the main objective - unsure if we will directly gain any contributors from this directly
* Lessons learned about the importance of content curation from the PQ Day sessions that we should apply to any event we (co)host - Hart will be discussing a co-lo event proposal with the GB today
* Actual presentations were a bit repetitive - the same issues/challenges/insights being discussed by multiple speakers  - indicative of where we may be able to provide focus
* Standards bodies seem to have conflicting requirements - bridging the gaps is one area where we could potentially engage
___
### Tooling WG and SonarCube CBOM plugin submission (Nicklas)
* WG creation approved 
* Nicklas Körtge designated as the interim leader
* cbomkit project
* cbomkit-theia project
* ACTION Kenny work with Nicklas on logistics
___
### Brian potential AWS tooling project

> * collegeag QKD quant key dist

Reviewer: Colleague
QKD = Quantum Key Distribution

> * collegeag QKD quant key dist

Reviewer 2: ```suggestion
* Colleague - QKD (Quantum Key Distribution)
```
